test/test_MDOP.jl

```diff
@@ -44,11 +41,14 @@ ss = zeros(length(s))
 [ss[i] = s[i][1] for i in 1:length(s)]
```

Comment:
`[ss[i] = s[i][1] for i in 1:length(s)]` is not a nice Julia syntax and should not be used for iteration, but only when you want to create a **new** array. Doing `[whatever for i in 1:N]` allocates a new array. In the current case, you should do instead:
```julia
for i in 1:length(s); ss[i] = s[i][1]; end
```
which is also one line.